server/app/com/xsn/explorer/databaseMigrations/BlockRewardsMigration.scala

```diff
@@ -0,0 +1,126 @@
+package com.xsn.explorer.databaseMigrations
+
+import com.alexitc.playsonify.core.FutureOr.Implicits.FutureOps
+import com.xsn.explorer.data.anorm.AnormPostgresDataHandler
+import com.xsn.explorer.data.anorm.dao.{BlockPostgresDAO, BlockRewardPostgresDAO}
+import com.xsn.explorer.data.async.BlockFutureDataHandler
+import com.xsn.explorer.executors.DatabaseExecutionContext
+import com.xsn.explorer.models.{BlockExtractionMethod, BlockRewards, rpc}
+import com.xsn.explorer.models.persisted.Block
+import com.xsn.explorer.models.values.{Blockhash, Height}
+import com.xsn.explorer.services.{BlockService, XSNService}
+import javax.inject.Inject
+import org.scalactic.{Bad, Good}
+import org.slf4j.LoggerFactory
+import play.api.db.Database
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success}
+
+class BlockRewardsMigration @Inject()(
```

Comment:
`MigrationRunner` should be enough, there is no need to keep renaming the file on each migration.